Makefile

```diff
@@ -138,15 +138,25 @@ endif
 endif
 
 # ---------------------------------------------------------------------
+# Electron Builder
+# ---------------------------------------------------------------------
+
+ELECTRON_BUILDER_OPTIONS=--$(TARGET_ARCH_ELECTRON_BUILDER) --extraMetadata.version=$(APPLICATION_VERSION)
```

Comment:
Huh? Maybe we're talking at cross purposes. What I was asking is whether changing:
```
ELECTRON_BUILDER_OPTIONS=--$(TARGET_ARCH_ELECTRON_BUILDER) --extraMetadata.version=$(APPLICATION_VERSION)
```
to
```
ELECTRON_BUILDER_OPTIONS = --$(TARGET_ARCH_ELECTRON_BUILDER) --extraMetadata.version=$(APPLICATION_VERSION)
```
would work?  (and then the same for the `ELECTRON_BUILDER_OPTIONS+=` lines further down)